### ISSUE TITLE
RCA Branding

### ISSUE THREAD

**Issue by lhstrh** [icyphy/lingua-franca#360]
Instead of continuing to refer our product as an Eclipse RCA, I think we should name it. Any proposals for a suitable name?

**Comment by edwardalee** [comment]
We could call the IDE "Frank, your Lingua Franca assistant". :-)

**Comment by lhstrh** [comment]
How about `Epoch`? Since the application is Eclipse-based, I think it makes sense to use an astronomy-related term. Many projects in the Eclipse ecosystem do this. Think of Theia, Equinox, etc. I think the clear reference to time is also fitting...

**Comment by cmnrd** [comment]
I like Epoch! I did a quick background check and it seems there are also no major name conflicts with other projects. The only relevant project I found is this one: http://epochjs.github.io/epoch/. It seems to be sufficiently distinct though.

BTW: This other epoch project looks really interesting. Its capability for drawing real-time charts could be relevant for Lingua Franca. I guess it could work really well for demonstration purposes, to visualize in real-time plots what the runtime is doing.
